src/Components/home/Calendar.jsx

```diff
@@ -25,16 +26,39 @@ const calendarHeader = {
 export function calendarCall(info, successCallback) {
   API.get(`/calendar?start=${info.start}&end=${info.end}`)
     .then((res) => successCallback(res.data))
-    .catch((err) => console.log(err));
+    .catch((err) => console.error(err));
 }
 
 export default function Calendar() {
+
   return (
     <FullCalendar
       defaultView="dayGridMonth"
-      plugins={[dayGridPlugin, timeGridPlugin]}
-      eventSources={[calendarCall, testEvents]}
+      eventTextColor="white"
+      eventBorderColor="rgba(0, 0, 0, 0)"
+      plugins={[dayGridPlugin, timeGridPlugin, googleCalendarPlugin, interactionPlugin]}
+      googleCalendarApiKey='AIzaSyBREtcqxmgY2SbJB5ddscMlKpWrHM0R1JM'
+      events={{googleCalendarId: "q2o6gp4ogd31rls2q7sqvl5nps@group.calendar.google.com"}}
```

Comment:
Should the API key and the calendar's static email address (I assume this is the static address) be public?

Reply:
I wasn't sure about this.  Here were my thoughts:
- Currently this is just a test calendar with a test API key, so I think having the API key public is fine
- The actual calendar API  key definitely needs to be private.  I'm not sure how to make it private yet, so that can be done in a future push.
- I'm not sure about the calendar's static email address.  That should be discussed.

Reply:
Fair enough. I will make a story to secure the API key before we transition to the public calendar.